### PR TITLE
Footer section (also fixes button color issue)

### LINKED ISSUE #9: Discord CTA & Footer

### PR DESCRIPTION
**Pull request by Singa-pirate**
Closes #9.

### CODE REVIEW

**Comment by Singa-pirate** [comment]
@nknguyenhc FYI, the links at the footer are currently not aligned with the links at the nav bar.
I designed it this way because in the future when the footer has sublinks, it will look nicer if they spread out the whole space.
image

File: new/src/components/button/button.scss [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.button {
+button[class^="button"] {
```
Comment (nknguyenhc) [inline new/src/components/button/button.scss:1]: Any bug if we leave it as `.button`?

Comment (Singa-pirate) [reply new/src/components/button/button.scss]: I faced issue due to a redundant css property defined in another file 🤣 
Changed back to using class name selectors, and made the discord button inherit both button classes' property

**Review by nknguyenhc — APPROVED** [review]
LGTM!

**Pull request merged**